lib/licensee/content_helper.rb

```diff
@@ -4,13 +4,14 @@
 module Licensee
   module ContentHelper
     DIGEST = Digest::SHA1
-    END_OF_TERMS_REGEX = /^\s*end of terms and conditions\s*$/i
+    END_OF_TERMS_REGEX = /^[\s#]*end of terms and conditions\s*$/i
```

Comment:
I wonder if it's worth catching markdown bold/italics here as well as header.

Reply:
Good call. 7bd1aa6.